src/main/features/claim/routes/amount.ts

```diff
@@ -23,10 +24,11 @@ function renderView (form: Form<ClaimAmountBreakdown>, res: express.Response): v
 
 function actionHandler (req: express.Request, res: express.Response, next: express.NextFunction): void {
   if (req.body.action) {
-    const form: Form<ClaimAmountBreakdown> = req.body
-    if (req.body.action.addRow) {
+    const form: ActionForm<ClaimAmountBreakdown> = req.body
+    if (form.action.addRow) {
       form.model.appendRow()
     }
+    form.postbackFocusTarget = ActionForm.buildPostbackFocusTarget(form.action)
```

Comment:
Instead of resigning it we could have `get postbackFocusTarget()` method that would have logic from `ActionForm.buildPostbackFocusTarget`.